modules/core/client/views/enroll.client.view.html

```diff
@@ -1,4 +1,5 @@
-<section data-ng-controller="EnrollController">
+<section data-ng-controller="EnrollController" id="content">
```

Comment:
!completed I couldn't put the sidebar into the index file without having it in the login screen. I did it this way in order to prevent that